### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Kavster92**
[DEMO LINK](https://Kavster92.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/CommentList/CommentList.tsx [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = (props) => {
+  const { comments } = props;
+
+  return (
+    <div className="CommentList">
```
Comment (nurdanokcu) [inline src/components/CommentList/CommentList.tsx:15]:
```suggestion
export const CommentList: React.FC<Props> = ({ comments }) => (
    <div className="CommentList">
```

It is only comments as props. So I believe this is shorter and more understandable.

File: src/components/PostInfo/PostInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+            <UserInfo user={user} />
+          )}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length > 0 ? (
```
Comment (nurdanokcu) [inline src/components/PostInfo/PostInfo.tsx:39]:
```suggestion
      {comments.length ? (
```

**Review by nurdanokcu — CHANGES_REQUESTED** [review]
Everything matches requirements. I added small suggestions . I will give it another look after the changes.

**Review by meforoma — APPROVED** [review]
✅

**Review by meforoma — APPROVED** [review]

**Review by meforoma — CHANGES_REQUESTED** [review]
test reject

**Review by meforoma — APPROVED** [review]

**Pull request closed without merging**